Update run-slam-cartographer.md (#980)
viamrobotics/docs: docs/services/slam/run-slam-cartographer.md

Because of this, Cartographer assumes your Rplidar will remain at roughly the same height while in use.
If maps are not building the way you expect, make sure your Rplidar is secure and at roughly the same height throughout the run.

### Issue: Offline mode produces an error after restart

If there is a saved map in `data_dir/map` and saved data in `data_dir/data` from a previous run, then offline mode
may error at startup, since the data has already been incorporated into the map. If that occurs, you can

> either clear `data_dir/map` to rerun offline mode, or clear `data_dir/data` to view the saved map.

Reviewer: Clearing the data directory does not work for this, as cartographer will throw the error `terminating with uncaught exception of type std::runtime_error: not enough data in data directory`
```suggestion
clear `data_dir/map` to rerun the dataset in offline mode.
```

Author: ah, thank you
